### PR TITLE
PC-942: Init

### PR DESCRIPTION
**Pull request by ulfryk**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@samwise-tech/forbidden-patterns",
+  "version": "0.0.0",
```
Comment (tomecko) [inline package.json]: ?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "description": "Simple tool to test files against forbidden patterns",
```
Comment (tomecko) [inline package.json]: :+1:

File: test/error-test.bats [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!./test/libs/bats/bin/bats
+
+@test "Should yeild errors when any of patterns match any of files" {
```
Comment (tomecko) [inline test/error-test.bats]: yield - typo

File: test/success-test.bats [hunk 4]
```diff
@@ -0,0 +1,18 @@
+#!./test/libs/bats/bin/bats
+
+@test "Should yield success when no occurances of patterns found" {
```
Comment (tomecko) [inline test/success-test.bats]: occurrences

File: package.json (hunk 1 above)
Comment (ulfryk) [reply package.json]: Still not releasable IMO

**Pull request merged**